js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
```

Comment:
components are repeated. As mentioned above, use Accordion from Chakra to avoid this